src/assets/i18n/de/translation.json

```diff
@@ -121,5 +121,13 @@
 				}
 			}
 		}
-	}
+	},
+  "signature-pad": {
```

Comment:
not correctly indented

Reply:
@TQjona , I have updated all translation files to use `spaces` instead of `tab` character.

Reply:
@ankitp-atharva The indention is controlled by prettier, which specifies tabs. Furthermore these files should be automatically formatted by the git commit hooks. Please troubleshoot the git commit hook: https://github.com/typicode/husky/tree/master#troubleshoot

Reply:
@TQjona , Fixed and updated